src/test/java/bad/robot/excel/workbook/PoiWorkbookTest.java

```diff
@@ -149,4 +154,16 @@ public void replaceThenLoadBlankCell() throws IOException {
         assertThat(adaptPoi(getCellForCoordinate(coordinate(A, 1), modified.workbook())), is(instanceOf(BlankCell.class)));
     }
 
-}
\ No newline at end of file
+    @Test
+    public void shouldCountMergedCellsCorrectly() throws IOException, InvalidFormatException {
+        // Create a workbook with a single empty sheet that has merged cells.
+        XSSFWorkbook workbook = new XSSFWorkbook();
+        XSSFSheet sheet = workbook.createSheet("Sheet1");
+        int firstMergedCol = 1;
+        int lastMergedCol = 5;
+        sheet.addMergedRegion(new CellRangeAddress(0, 0, firstMergedCol, lastMergedCol));
+
```

Comment:
Be nice to add an assertion here to ensure/demonstrate that the `addMergedRegion` did what it's supposed to. I was a bit confused how we should expect the underlying sheet to have changed just reading it cold.